src/apps/user-management/DeleteUser/DeleteUser.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+import * as Dialog from '@radix-ui/react-dialog';
+import { Trash, X } from '@phosphor-icons/react';
+import { Button } from '@components/Button';
+import type { PostgrestError } from '@supabase/supabase-js';
+import { removeUserFromProject } from '@backend/crud';
+import { supabase } from '@backend/supabaseBrowserClient';
+import type { Translations, UserProfile, ExtendedUserProfile } from 'src/Types';
+
+interface DeleteUserProps {
+  i18n: Translations;
+
+  me: UserProfile;
+
+  user: ExtendedUserProfile;
+
+  onDeleteUser(member: ExtendedUserProfile): void;
+
+  onDeleteError(error: PostgrestError): void;
+}
+
+export const DeleteUser = (props: DeleteUserProps) => {
+  const { t } = props.i18n;
+
+  const { user } = props;
+
+  const { nickname, first_name, last_name } = user;
+
+  const isMe = props.me.id === user.id;
+
+  const [busy, setBusy] = useState(false);
+
+  const name = nickname
+    ? nickname
+    : first_name || last_name
+    ? [first_name, last_name].join(' ')
+    : undefined;
+
+  const onDelete = () => {
+    setBusy(true);
+
+    removeUserFromProject(supabase, user.id, user.org_group_id).then(
```

Comment:
This doesn't really do anything on the project level, correct? Worth renaming this helper method to `removeUserFromGroup` instead?